docs/specs/xref.yml

```diff
@@ -654,3 +654,31 @@ inputs:
 outputs:
   build.log: |
     ["error","schema-not-found","Unknown schema ''","docs/a.json"]
+---
+# support <xref href="uid"></xref>
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    str <xref href="c"></xref>
+    str <xref href="d" data-raw-source="@mention"></xref>
+    str <xref href="d" data-throw-if-not-resolved="False" data-raw-html="@higher" data-raw-source="@mention"></xref>
+    some text
+    str <xref href="d" data-raw-html="@higher" data-raw-source="@mention"></xref>
+    multiple xref <xref href="e" data-raw-source="@mentione"></xref> some text <xref href="f" data-raw-source="@mentionf"></xref>
+    multiple xref <xref href="e" data-raw-source="@mentione"></xref><xref href="f" data-raw-source="@mentionf"></xref>
+  docs/c.md: |
+    ---
+    title: Title from yaml header c
+    uid: c
+    ---
+outputs:
+  docs/a.json: |
+    {"content" : "<p>str <a href='c'>Title from yaml header c</a>\nstr @mention\nstr @higher\nsome text\nstr @higher\nmultiple xref @mentione some text @mentionf\nmultiple xref @mentione@mentionf</p>\n"}
+  docs/c.json:
+  build.log: |
+    ["warning","uid-not-found","Cannot find uid 'd' using xref '<xref href=\"d\" data-raw-source=\"@mention\">'","docs/a.md",2,12]
+    ["warning","uid-not-found","Cannot find uid 'd' using xref '<xref href=\"d\" data-raw-html=\"@higher\" data-raw-source=\"@mention\">'","docs/a.md",5,12]
+    ["warning","uid-not-found","Cannot find uid 'e' using xref '<xref href=\"e\" data-raw-source=\"@mentione\">'","docs/a.md",6,12]
+    ["warning","uid-not-found","Cannot find uid 'f' using xref '<xref href=\"f\" data-raw-source=\"@mentionf\">'","docs/a.md",6,12]
+    ["warning","uid-not-found","Cannot find uid 'e' using xref '<xref href=\"e\" data-raw-source=\"@mentione\">'","docs/a.md",7,12]
+    ["warning","uid-not-found","Cannot find uid 'f' using xref '<xref href=\"f\" data-raw-source=\"@mentionf\">'","docs/a.md",7,12]
```

Comment:
column is 12 for both `e` and `f`...

Reply:
Yes, do not see a good way to get the proper column number. And also while the HTML block has multiple lines, just use the starting line since there is not an accurate way to get the line number within the block.